src/modules/multilevelFacets/MultilevelFacetsWrapper.js

```diff
@@ -20,6 +21,7 @@ const MultilevelFacetsWrapper = (props) => {
             FacetItemComponent={FacetItemComponent}
             label={label}
             collapsible={collapsible}
+            minViewMore={minViewMore || 3}
```

Comment:
Can we add `minViewMore` to defaultProps.

Reply:
need more clarity

Reply:
We can initialise minViewMore to a default value. [ref](https://reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values)

Reply:
got it

Reply:
done